src/Features/Core/Portable/EditAndContinue/ManagedEditAndContinueDebuggerServiceExtensions_REMOVE.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Debugger.Contracts.EditAndContinue;
+
+namespace Microsoft.CodeAnalysis.EditAndContinue
+{
+    internal static class ManagedEditAndContinueDebuggerServiceExtensions_REMOVE
+    {
+        /// <summary>
+        /// This will be removed when IManagedEditAndContinueDebuggerService gets the method for real
+        /// </summary>
+        public static ValueTask<string> GetCapabilitiesAsync(this IManagedEditAndContinueDebuggerService _1, CancellationToken _2)
```

Comment:
>string [](http://example.com/codeflow?start=32&length=6)

The API actually returns `ImmutableArray<string>` (we changed it). It's already split by ` `.